app/helpers/mailbox_helper.rb

```diff
@@ -2,6 +2,6 @@ module MailboxHelper
   def unread_messages_count
     # how to get the number of unread messages for the current user
     # using mailboxer
-    mailbox.inbox(unread: true).count(:id, distinct: true)
+    mailbox.inbox(unread: true).count(:id)
```

Comment:
Ok maybe this is a Rails 4 vs. Rails 5 thing. Another reason why your cucumber tests need to make sure the user can see the unread message count - to make sure this method still works.